src/Microsoft.ML.Core/Utilities/ReservoirSampler.cs

```diff
@@ -47,7 +47,7 @@ public interface IReservoirSampler<T>
     /// This class produces a sample without replacement from a stream of data of type <typeparamref name="T"/>. 
     /// It is instantiated with a delegate that gets the next data point, and builds a reservoir in one pass by calling <see cref="Sample"/> 
     /// for every data point in the stream. In case the next data point does not get 'picked' into the reservoir, the delegate is not invoked.
-    /// Sampling is done according to the algorithm in this paper: <see href="http://epubs.siam.org/doi/pdf/10.1137/1.9781611972740.53"/>.
+    /// Sampling is done according to the algorithm in this paper: <a href="http://epubs.siam.org/doi/pdf/10.1137/1.9781611972740.53">http://epubs.siam.org/doi/pdf/10.1137/1.9781611972740.53</a>.
```

Comment:
❓ Why make this change? I would expect one of the following:

1. The former case (`href` specified but no explicit text)
2. The form `<see href="http://epubs.siam.org/doi/pdf/10.1137/1.9781611972740.53">Reservoir-base Random Sampling with Replacement from Data Stream</a>`
3. The form `<see href="http://epubs.siam.org/doi/pdf/10.1137/1.9781611972740.53">Reservoir-base Random Sampling with Replacement from Data Stream (PDF, Proceedings of the 2004 SIAM International Conference on Data Mining)</a>`

:memo: `<see href` is a well-supported form for external links in documentation comments, more so than `<a href`.

Reply:
Doesn't seem see contains any href:
[https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/see](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/see)

And this was a bit unsettling:
[https://stackoverflow.com/questions/6960426/c-sharp-xml-documentation-website-link](https://stackoverflow.com/questions/6960426/c-sharp-xml-documentation-website-link)

---
In reply to: [202773275](https://github.com/dotnet/machinelearning/pull/529#discussion_r202773275) [](ancestors = 202773275)